### PR TITLE
test: add test case

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | test: add test case |
| 🇨🇳 Chinese | 无需纳入ChangeLog |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/float-button/index.zh-CN.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ demo:
 | onClick | 点击按钮时的回调 | (event) => void | - |  |
 | href | 点击跳转的地址，指定此属性 button 的行为和 a 链接一致 | string | - |  |
 | target | 相当于 a 标签的 target 属性，href 存在时生效 | string | - |  |
-| badge | 带圆形徽标数字的悬浮按钮 | 参考[badge](/components/badge-cn#api) | - | 5.4.0 |
+| badge | 带圆形徽标数字的悬浮按钮 | 参考[badge](/components/badge-cn#api)（只支持部分属性） | - | 5.4.0 |
```
Comment (afc163) [inline components/float-button/index.zh-CN.md:50]: 这个很奇怪，具体是哪些属性呢

Comment (li-jia-nan) [reply components/float-button/index.zh-CN.md:50]: image

Comment (li-jia-nan) [reply components/float-button/index.zh-CN.md:50]: image

Comment (li-jia-nan) [reply components/float-button/index.zh-CN.md:50]: 本来放个链接，ref 到 ts 定义那里，但是现在 master 没有代码，只能先 link 到 feature 去，和闲夕大佬沟通了一下，他觉得先不放链接了

File: components/float-button/index.en-US.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ FloatButton. Available since `5.0.0`.
 | onClick | Set the handler to handle `click` event | (event) => void | - |  |
 | href | The target of hyperlink | string | - |  |
 | target | Specifies where to display the linked URL | string | - |  |
-| badge | float-button with round logo number | refer [badge](/components/badge#api) | - | 5.4.0 |
+| badge | float-button with round logo number | refer [badge](/components/badge#api)(only a part attributes are supported) | - | 5.4.0 |
```
Comment (MadCcc) [inline components/float-button/index.en-US.md:49]:
```suggestion
| badge | Attach Badge to FloatButton. `status` `size` and other props related are not supported. | [BadgeProps](/components/badge#api)  | - | 5.4.0 |
```

File: components/float-button/index.zh-CN.md (hunk 1 above)
Comment (MadCcc) [inline components/float-button/index.zh-CN.md:50]:
```suggestion
| badge | 带徽标数字的悬浮按钮（不支持 `status` `size` 以及相关属性） | [BadgeProps](/components/badge-cn#api) | - | 5.4.0 |
```

**Comment by MadCcc** [comment]
`size` 其实可以考虑支持一下，只有 status 是个变体

**Comment by li-jia-nan** [comment]
> `size` 其实可以考虑支持一下，只有 `status` 是个变体

需要设计师支持一下吗？

File: components/float-button/index.zh-CN.md (hunk 1 above)
Comment (afc163) [reply components/float-button/index.zh-CN.md]: 可以按 `y` 把 github 文件地址固定一下。

**Comment by MadCcc** [comment]
> > `size` 其实可以考虑支持一下，只有 `status` 是个变体
> 
> 需要设计师支持一下吗？

按照现有设计就可以

**Review by MadCcc — APPROVED** [review]

**Pull request merged**